frontend/ui/widget/keyvaluepage.lua

```diff
@@ -134,7 +134,8 @@ function KeyValueItem:init()
         else
             -- Both can fit: break the 1/2 widths
             if self.value_align == "right" or self.value_overflow_align == "right_always"
-                    or (self.value_overflow_align == "right" and value_w_rendered > value_w) then
+                    or (self.value_overflow_align == "right" and value_w_rendered > value_w)
+                    or key_w_rendered < key_w then -- it's the value that can't fit (longer), this way it stays closest to border
```

Comment:
Isn't `key_w_rendered < key_w` a too simple test ? I mean, it would catch small keys and values when 50/50 and get the value mostly always right aligned ?
That's my thought just reading this. But testing the PR, I don't see that happening :) Why is it not happening ? :/

Reply:
Haha, that's because the ones that need not adjust (fit the specific ratio) don't come here at all.